HTTPRequest/HTTPRequest.DotNet.cs

```diff
@@ -0,0 +1,192 @@
+﻿// IF WE ARE NOT USING UNITY OR WE HAVE DEFINED THE
+// MAGE_USE_WEBREQUEST MACROS, USE THIS VERSION
+#if !UNITY_5 || MAGE_USE_WEBREQUEST
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Net;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+
+public class HTTPRequest {
```

Comment:
Formatting is a little off here, I guess the switch to Windows (VS is default spaces, Tools -> Options -> Text Editor -> All Languages -> Tabs)